tests/mongo.yml

```
@@ -56,26 +56,35 @@ args:
   - mongo/jstests/core/ddl/convert_to_capped.js
   - mongo/jstests/noPassthrough/noncapped_oplog_creation.js
   - mongo/jstests/noPassthroughWithMongod/capped4.js
+  - mongo/jstests/aggregation/sources/collStats/query_exec_stats.js
+  - mongo/jstests/core/notablescan_capped.js
+  - mongo/jstests/core/local_tail_capped.js
+  - mongo/jstests/core/query/awaitdata_getmore_cmd.js
+  - mongo/jstests/core/query/cursor/getmore_cmd_maxtimems.js
+  - mongo/jstests/core/query/cursor/tailable_cursor_invalidation.js
+  - mongo/jstests/core/query/cursor/tailable_getmore_batch_size.js
+  - mongo/jstests/core/query/cursor/tailable_skip_limit.js
+  - mongo/jstests/core/write/crud_api.js
 
 results:
   includes:
     aggregation:
-      - mongo/jstests/aggregation/ifnull.js # unhandled element type `Undefined (value) Deprecated`
+      - mongo/jstests/aggregation/ifnull.js # https://github.com/FerretDB/FerretDB/issues/2286
```

```suggestion
      # https://github.com/FerretDB/FerretDB/issues/2286
      - mongo/jstests/aggregation/ifnull.js
```

To not use two comment styles at the same time